### PR TITLE
Second section

### PR DESCRIPTION
**Pull request by rpire**
### Added the "Works" and the "About Myself" sections.

Files changed:

- index.html
- style.css
- Added the required images to the "Media" folder.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div>
```
Comment (arslanbisharat) [inline index.html:48]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 2]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
```
Comment (arslanbisharat) [inline index.html:62]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 3]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
```
Comment (arslanbisharat) [inline index.html:76]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 4]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
```
Comment (arslanbisharat) [inline index.html:90]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 5]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
```
Comment (arslanbisharat) [inline index.html:104]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 6]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+    </div>
```
Comment (arslanbisharat) [inline index.html:120]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 7]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+  <section class="works">
+    <h2>Projects</h2>
+    <img class="works-img" src="./Media/work-img-1.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
```
Comment (arslanbisharat) [inline index.html:41]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 8]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+      </a>
+    </div><img class="works-img" src="./Media/work-img-3.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
```
Comment (arslanbisharat) [inline index.html:70]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 9]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+      </a>
+    </div><img class="works-img" src="./Media/work-img-4.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
```
Comment (arslanbisharat) [inline index.html:84]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 10]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+      </a>
+    </div><img class="works-img" src="./Media/work-img-5.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
```
Comment (arslanbisharat) [inline index.html:98]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 11]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+      </a>
+    </div>
+    <img class="works-img" src="./Media/work-img-6.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
```
Comment (arslanbisharat) [inline index.html:113]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 12]
```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
...
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+    veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
+    eiusmod
+  </p>
+  <div class="about-container">
+    <div class="cv-box">
+      <span>Get my CV</span>
+      <div></div>
+    </div>
```
Comment (arslanbisharat) [inline index.html:134]: - [ ] This should be a button and The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 2 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 3 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 4 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 5 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 8 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 9 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 10 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 11 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 12 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 6 above)
Comment (rpire) [reply index.html]: Satisfied.

File: index.html (hunk 7 above)
Comment (rpire) [reply index.html]: Satisfied.

**Comment by rpire** [comment]
## Hello @arslanbisharat !

Thanks for reviewing my work.

Changes specified in the request have been implemented:

- All projects in the "works" section were presented as `li` elements in a `ul` mother element.
- All technologies (HTML, CSS, JavaScript...) were presented as `li` elements in a `ul` mother element.
- The "Get my CV" button was enclosed between `a` and `/a` tags.

Thanks again for reviewing my work, happy coding! 🤓🚀

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi @rpire , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please try to add some margin for each project of the `work` section like you did in the `about` section.


Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
Hello @arslanbisharat !

Thank you for reviewing my code, it's always good to be approved 🙌🎉

I would love to implement your recommendation about the margin for the "works" section but I believe that doing so would change the appearance of the webpage and prevent it from being an exact copy of the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095).

Please let me know if it wouldn't matter to change the design a little bit by applying the requested margins. If you think that there would be some kind of inconvenience by doing so I will merge just the way the project is right now.

Thanks again for reviewing my work. 🤓

Cheers and happy coding! 🚀

**Comment by hemant-soni-vst-au4** [comment]
- [ ] Yes go ahead and merge the PR just the way project is right now.

Cheers and Happy Coding! 🚀

**Pull request merged**